src/Npc.js

```diff
@@ -99,6 +99,27 @@ class Npc extends Character {
       this.addItem(newItem);
     });
 
+    // Hydrate equipment
+    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
+      const eqDefs = this.defaultEquipment;
+      this.defaultEquipment = new Map();
+      for (const slot in eqDefs) {
+        const itemDef = eqDefs[slot];
+        try {
+          let newItem = state.ItemFactory.create(state.AreaManager.getAreaByReference(itemDef), itemDef);
+          newItem.initializeInventory(itemDef.inventory);
+          newItem.hydrate(state);
+          state.ItemManager.add(newItem);
+          newItem.isEquipped = true;
+          this.equip(newItem, newItem.metadata.slot);
```

Comment:
You don't need to use `metadata.slot` here. 1) an item may have more than one valid slot so you can't trust this to be a single string, 2) You already have the `slot` available from the key (described above in the loop change)

Reply:
hmmm, that slot wasn't doing it for me.  It was containing the index of eqDefs, so removing the eqDefs from the loop fixes that I take it?